server/src/test/java/org/opensearch/rest/action/admin/cluster/RestClusterStatsActionTests.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.rest.action.admin.cluster;
+
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.test.OpenSearchTestCase;
+import org.opensearch.test.rest.FakeRestRequest;
+
+import java.util.HashMap;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.object.HasToString.hasToString;
+import static org.mockito.Mockito.mock;
+
+public class RestClusterStatsActionTests extends OpenSearchTestCase {
```

Comment:
Can we have one success case ?

Reply:
OK. I think with properly inserted mocks you should be able to verify. Lets create a follow-up PR for this. Can you add the manual testing details to PR ?

Reply:
Extracted the logic for cluster stats request computation to a static block to make it visible for testing.